HEC.FDA.ModelTest/unittests/StageDamageShould.cs

```diff
@@ -256,32 +256,13 @@ private Inventory CreateInventory()
             return inventory;
         }
 
-
-        [Fact]
```

Comment:
do we have a similar test somewhere?

Reply:
not necessarily. Truth is this test hadn't been run in a long time, and the case it was testing throws an exception way before it looks like it did in this test. A lot of processing is kicked off right in the constructor of IASD. We might reconsider that, because constructors can't fail as gracefully as we can on other methods. 

For now, this test just isn't useful, and should go.